db/migrate/20131209063832_fix_bad_markdown.rb

```diff
@@ -1,4 +1,10 @@
 class FixBadMarkdown < ActiveRecord::Migration
+  class LegacyProblem < ActiveRecord::Base
+    self.primary_key = "problem_set_id"
+
+    self.table_name = "problem_sets_problems"
+  end
```

Comment:
I'm pretty sure that in this migration it should just be `class Problem < ActiveRecord::Base; end`.

I haven't managed to test the other migration yet (need to do it when there are problems in the DB).

But wouldn't it be much simpler to just comment out the code, since our production DB is already shipshape and we don't have other DBs that need to be migrated?

I spotted another migration that may be problematic: 20131217112526_change_test_set_and_test_case_sample_and_prerequisites.rb